lib/ioki/apis/passenger_api.rb

```diff
@@ -138,11 +138,22 @@ class PassengerApi
         model_class:          Ioki::Model::Passenger::LogpayUrl,
         outgoing_model_class: Ioki::Model::Passenger::LogpayPaymentMethod
       ),
+      Endpoints::Create.new(
+        :logpay_paypal_client_token,
+        base_path:   [API_BASE_PATH],
+        path:        ['logpay', 'paypal_client_token'],
+        model_class: Ioki::Model::Passenger::LogpayPaypalClientToken
+      ),
       Endpoints::Index.new(
         :payment_methods,
         base_path:   [API_BASE_PATH],
         model_class: Ioki::Model::Passenger::PaymentMethod
       ),
+      Endpoints::Create.new(
+        :payment_method,
+        base_path:   [API_BASE_PATH],
+        model_class: Ioki::Model::Passenger::PaymentMethod
+      ),
```

Comment:
I'd expect a test for this in https://github.com/ioki-mobility/ioki-ruby/blob/a6ca93f8f5f222b451eb9fd0b083aa08844c418d/spec/ioki/passenger_api_spec.rb#L308

Reply:
Added!